src/airgap/airgap_input.txt

```diff
@@ -108,13 +108,6 @@
     aqua-security-job-runner
     wiz-job-runner
 
-[ff]:
-    ff-postgres-migration-signed
```

Comment:
Can we confirm there are no FF only images in some other module-like platform? This can be done by creating the images.txt and comparing.